### PR TITLE
[CORE-10420] add operator/manifest tabs to configure-cni-plugins page

### PR DESCRIPTION
**Pull request by aaaaaaaalex**
Product Version(s):
OSS 3.28 / EE 3.19 EP2

SME review:
- [ ] An SME has approved this change. 

DOCS review:
- [ ] A member of the docs team has approved this change.

Merge checklist:

- [ ] Deploy preview inspected wherever changes were made 
- [ ] Build completed successfully
- [ ] Test have passed

### CODE REVIEW

**Comment by aaaaaaaalex** [comment]
Just noticed this is PR #1492 - truly a historic PR

**in the PR of 1492,
Alex added an operator how-to**

File: calico-enterprise/reference/component-resources/configuration.mdx [hunk 1]
```diff
@@ -241,18 +265,18 @@ Example CNI config:
...
   }
 }
 ```
 
-The {{prodname}} CNI plugin uses {{prodname}}'s `endpoint-status` directory to determine when the dataplane has been programmed for a pod.
-By default, the {{prodname}} CNI plugin will look for the directory at: `/var/run/calico/endpoint-status`.
-To enable and adjust where {{prodname}} writes `endpoint-status`, the `endpointStatusPathPrefix` option must be configured for [Felix](node/felix/configuration.mdx).
+The {{prodname}} CNI plugin reads Felix's `endpoint-status` directory to determine when the dataplane has been programmed for a pod.
+If left unset, the {{prodname}} CNI plugin will look for the directory at `/var/run/calico/endpoint-status`. The path `/var/run/calico` is commonly mounted to the {{prodname}} DaemonSet, meaning it can be written to by the Felix container, and read by the (host-namespace) {{prodname}} CNI plugin.
+To enable the `endpoint-status` directory, and adjust which directory of the Felix container it is written to, the `endpointStatusPathPrefix` option must be configured for [Felix](node//felix/configuration.mdx).
```
Comment (ctauchen) [inline calico-enterprise/reference/component-resources/configuration.mdx]:
```suggestion
To enable the `endpoint-status` directory, and adjust which directory of the Felix container it is written to, the `endpointStatusPathPrefix` option must be configured for [Felix](node/felix/configuration.mdx).
```

File: calico-enterprise/reference/component-resources/configuration.mdx [hunk 2]
```diff
@@ -224,15 +224,39 @@ you must also run `{{imageNames.kubeControllers}}` with the policy, profile, and
 ```
 
 When using `type: k8s`, the {{prodname}} CNI plugin requires read-only Kubernetes API access to the `Pods` resource in all namespaces.
+</TabItem>
+</Tabs>
 
 ### Enabling policy setup timeout
 
-The `policy_setup_timeout_seconds` option makes the {{prodname}} CNI plugin wait to start a new pod until one of the following conditions occurs:
+The {{prodname}} CNI plugin can be configured to delay new pods from starting their containers until one of the following conditions occurs:
```
Comment (ctauchen) [inline calico-enterprise/reference/component-resources/configuration.mdx]: s/delay/prevent

"delay THING from VERBING" doesn't sound right, does it?

**Review by ctauchen — COMMENTED** [review]
Small nits, apply globally. Otherwise LGTM.

**Comment by ctauchen** [comment]
LGTM

**Pull request merged**